src/twisted/web/http.py

```diff
@@ -1416,7 +1416,15 @@ def _sanitize(val):
             cookie = cookie + b"; HttpOnly"
         if sameSite:
             sameSite = _ensureBytes(sameSite).lower()
-            if sameSite not in [b"lax", b"strict"]:
+            # See more info about sameSite usage here
+            # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#samesitesamesite-value
+            if not secure and sameSite == b"none":
+                raise ValueError(
+                    "Invalid value for sameSite: "
+                    + repr(sameSite)
+                    + '. Missing the "secure" attribute'
+                )
+            if sameSite not in [b"lax", b"strict", b"none"]:
```

Comment:
The API docs need to be updated to describe the new allowed value

Reply:
True. Thanks.

Here is a suggestion for the docstring from line 1365 , as I can't add an inline comment or suggestion

```
        @param sameSite: One of L{None} (default), C{'lax'}, C{'none'} or C{'strict'}.
            Direct browsers not to send this cookie on cross-origin requests.
            See:
            U{https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#samesitesamesite-value}
        @type sameSite: L{None}, L{bytes} or L{str}
```

Reply:
> @type sameSite: L{None}, L{bytes} or L{str}

Minor nitpick: wherever possible, let's use type annotations rather than `@type` epytext fields; it's more compact, more useful for more tools, and conveys the same information more precisely.